docs/_docs/liquid/tags.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Tags Filters
+permalink: "/docs/liquid/tags/"
+---
+All of the standard Liquid
+[tags](https://shopify.github.io/liquid/tags/control-flow/) are supported.
+Jekyll has a few built in tags to help you built your site. You can also create
+your own tags using [plugins](/docs/plugins/).
+
+## Includes
+
+If you have page snippets that you use repeatedly across your site an
+[include](/docs/includes/) is the perfect way make this more maintainable.
+
+## Code snippet highlighting
+
+Jekyll has built in support for syntax highlighting of over 60 languages
+thanks to [Rouge](http://rouge.jneen.net). Rouge is the default highlighter
+in Jekyll 3 and above. To use it in Jekyll 2, set `highlighter` to `rouge`
+and ensure the `rouge` gem is installed properly.
+
+Alternatively, you can use [Pygments](http://pygments.org) to highlight
```

Comment:
would be nice to have a warning stating that pygments is unsupported in jekyll 4 here

Reply:
Agree with @ashmaroli on this one. Let's wait until Jekyll 4 is ready before mentioning it in the docs. 

https://github.com/jekyll/jekyll/pull/7205/files/2ee8ff8da65a4dfe4bdcebc56685cda49caead5e..6273f95d2535e1d0972a2ac97515ff84c0892372#diff-8f5f2871b9faa35c84659bc37cd746cc